app/models/concerns/solr_holdings.rb

```diff
@@ -13,11 +13,17 @@ def mhld
   end
 
   def items
-    return [] if self[:item_display].blank?
+    return [] if self[:item_display].blank? && self[:item_display_struct].blank?
 
-    @items ||= self[:item_display].map do |item_display|
+    @items ||= self[:item_display_struct]&.map do |item_display|
       Holdings::Item.new(item_display, document: self)
-    end.sort_by(&:full_shelfkey)
+    end&.sort_by(&:full_shelfkey)
+
+    @items ||= self[:item_display]&.map do |item_display|
```

Comment:
Is this just a fallback until the index is fully repopulated?  Can you leave a comment about that if this is the case?